js/plates-model/crust.ts

```diff
@@ -207,18 +207,20 @@ export default class Crust {
         { rock: Rock.Gabbro, thickness: thickness * 0.7 }
       ];
     } else if (fieldType === "continent") {
-      let continentalSediment: Rock;
+      let continentalCrust: Rock;
       const range = BASE_CONTINENTAL_CRUST_THICKNESS - BASE_OCEANIC_CRUST_THICKNESS;
       if (thickness < BASE_OCEANIC_CRUST_THICKNESS + 0.4 * range) {
-        continentalSediment = Rock.Limestone;
+        continentalCrust = Rock.Limestone;
       } else if (thickness < BASE_OCEANIC_CRUST_THICKNESS + 0.8 * range) {
-        continentalSediment = Rock.Shale;
+        continentalCrust = Rock.Shale;
       } else {
-        continentalSediment = Rock.Sandstone;
+        continentalCrust = Rock.Sandstone;
       }
+      const granite = Math.max(0, thickness - SHALE_LIMESTONE_SANDSTONE_THICKNESS - MAX_REGULAR_SEDIMENT_THICKNESS);
       this.rockLayers = [
-        { rock: continentalSediment, thickness: SHALE_LIMESTONE_SANDSTONE_THICKNESS },
-        { rock: Rock.Granite, thickness: thickness - SHALE_LIMESTONE_SANDSTONE_THICKNESS }
+        { rock: Rock.ContinentalSediment, thickness: MAX_REGULAR_SEDIMENT_THICKNESS },
```

Comment:
PT story says "about as much as the oceanic sediment in the oceans" so we reuse the same sediment thickness constant.

Reply:
I'd still set `canSubduct` to `false`. This property decides whether crust/field can subduct or not. And we assume that only oceanic fields can subduct. That's why only oceanic rocks have this property set to true.

In other words, you could think that canSubduct is a property that will decide whether a crust is oceanic or continental. The name is a bit more generic tho, as we also have islands, etc. Continental sediment is part of a continental crust.